lib/house.rb

```diff
@@ -0,0 +1,66 @@
+class House
+  attr_reader :address, :rooms
+
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+
+  end
+
+  def price
+    if @price[0] == "$"
+      @price[1..-1].to_i
+    elsif (@price[0] != "$") && (@price.class == String)
+      @price.to_i
+    elsif @price.class == Integer
+      @price
+    else
+    end
+  end
+
+  def add_room(room)
+    @rooms << room
+  end
+
+  def above_market_average?
+    return true if price > 500000
```

Comment:
This works, but you can even simplify this to  be `price > 500000` as that will return a boolean based on the condition.